app/models/batch_context.rb

```diff
@@ -180,7 +180,7 @@ def verify_output_dir_no_exists
   def verify_file_manifest_selected_for_media
     return unless content_structure == 'media' && !using_file_manifest
 
-    errors.add(:content_structure, 'requires a file manifest.  Please select the checkbox and ensure a file manifest is present.')
+    errors.add(:content_structure, 'requires a file manifest.  Please indicate you have a file manifest and ensure a file manifest is present.')
```

Comment:
fixes an existing error message, this control is no longer a checkbox, it is a radio button